js-wixcode-sdk/$w/VideoBox.service.json

```diff
@@ -0,0 +1,608 @@
+        "docs":
+          { "summary": "Pauses playback.",
+            "description":
+              [ "The `pause()` function pauses the current video and fires a `pause` event.",
+                "",
+                " The `pause` event handlers set on the video box by the [onPause( )](#onPause)",
+                " function or in the Editor are called." ],
+            "links": [],
+            "examples":
+              [ { "title": "Pause playback",
+                  "body":
+                    [ "$w(\"#myVideoBox\").pause();" ],
+                  "extra":
+                    {  } },
+                { "title": "Pause playback and log a message when done",
+                  "body":
+                    [ "$w(\"#myVideoBox\").pause()",
+                      "  .then( () => {",
+                      "    console.log(\"Done with pause\");",
+                      "  } );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } },
+      { "name": "play",
+        "labels": [],
+        "nameParams": [],
+        "params": [],
+        "ret":
+          { "type":
+              { "name": "Promise",
+                "typeParams":
+                  [ "void" ] },
+            "doc": "Fulfilled - When playback begins or resumes." },
+        "locations":
+          [ { "lineno": 233,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Begins or resumes playback.",
+            "description":
+              [ "The `play()` function plays the current video and fires a `play` event.",
+                "",
+                " The `play` event handlers that are set on the video box either by the [onPlay( )](#onPlay)",
+                " function or in the Editor are called." ],
+            "links": [],
+            "examples":
+              [ { "title": "Begin or resume playback",
+                  "body":
+                    [ "$w(\"#myVideoBox\").play();" ],
+                  "extra":
+                    {  } },
+                { "title": "Begin or resume playback and log a message when done",
+                  "body":
+                    [ "$w(\"#myVideoBox\").play()",
+                      "  .then( () => {",
+                      "    console.log(\"Done with play\");",
+                      "  } );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } },
+      { "name": "stop",
+        "labels": [],
+        "nameParams": [],
+        "params": [],
+        "ret":
+          { "type":
+              { "name": "Promise",
+                "typeParams":
+                  [ "void" ] },
+            "doc": "Fulfilled - When playback has been stopped." },
+        "locations":
+          [ { "lineno": 287,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Pauses playback.",
```

Comment:
This is not correct if the description is correct since it moves to the beginning of the video. 
Maybe write "Stops the playback"